src/shared/components/magasinet-card/magasinet-card.css

```diff
@@ -0,0 +1,41 @@
+/**
+ * Block
+ */
+
+.hw-card-magasinet {
+  position: relative;
+  background: var(--hw-color-white);
+  padding: var(--hw-spacing-large) var(--hw-spacing-larger);
+  text-align: center;
+  border: 3px solid var(--hw-color-gray-light);
+
+&__overlay {
+   opacity: 1;
+   background-color: #ffffff;
+&:hover {
+   opacity: 0.8;
+ }
+}
+
+&__title {
+   margin-bottom: var(--hw-spacing-small);
+   transition: color 0.5s ease(0.165, 0.84, 0.44, 1);
+ }
+
+&__sub-title {
+   margin-bottom: var(--hw-spacing-large);
+ }
+
+&__text {
+   margin-top: var(--hw-spacing-larger);
+   margin-bottom: var(--hw-spacing-large);
+ }
+
+&__hr {
```

Comment:
We should use our existing hr
https://hedwig-docs.herokuapp.com/bring.html#/Hr?a=regular